common/src/main/java/com/wynntils/commands/FeatureCommand.java

```diff
@@ -51,17 +55,54 @@ private LiteralCommandNode<CommandSourceStack> buildListNode() {
     }
 
     private int listFeatures(CommandContext<CommandSourceStack> context) {
-        Set<Feature> features = FeatureRegistry.getFeatures().stream().collect(Collectors.toUnmodifiableSet());
+        List<Feature> features = FeatureRegistry.getFeatures().stream()
+                .filter(feature -> !(feature instanceof DebugFeature) || WynntilsMod.isDevelopmentEnvironment())
+                .sorted(Feature::compareTo)
+                .toList();
 
         MutableComponent response = new TextComponent("Currently registered features:").withStyle(ChatFormatting.AQUA);
 
+        String lastCategory = null;
+
         for (Feature feature : features) {
+            Class<?> superclass = feature.getClass().getSuperclass();
+
+            ChatFormatting color = ChatFormatting.WHITE;
+            String translatedName = feature.getTranslatedName();
+
+            if (superclass == UserFeature.class || superclass == StateManagedFeature.class) {
+                if (feature.isEnabled()) {
+                    color = ChatFormatting.GREEN;
+                } else {
+                    color = ChatFormatting.RED;
+                }
+            } else if (superclass == DebugFeature.class) {
+                color = ChatFormatting.YELLOW;
+
+                if (feature.isEnabled()) {
+                    translatedName += " {ENABLED DEBUG}";
+                } else {
+                    translatedName += " {DISABLED DEBUG}";
+                }
+            }
+
+            System.out.println("feature.getCategory() = " + feature.getCategory());
+
+            if (!Objects.equals(lastCategory, feature.getCategory())) {
```

Comment:
I think this would break if we *only* had uncategorized features. :) This is more an academic question, of course, but the warning bell triggered in the back of my head. Maybe just add a comment `// this assumes we have at least one categorized feature` to show the intent.

Reply:
Or, yeah, maybe not breaks. We would not get a category header at all. Which would probably be correct. I dunno. Just ignore my ranting, I guess.

Reply:
Also, debug line. Oops. 

And no, it would not. Default is null, uncategorized is "". So, it would trigger.

Reply:
Ah, I thought uncategorized were null.  Good.